tools/talis/add.go

```diff
@@ -0,0 +1,55 @@
+package main
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/spf13/cobra"
+)
+
+func addCmd() *cobra.Command {
+	var (
+		rootDir  string
+		count    int
+		nodeType string
+		region   string
+	)
+	cmd := &cobra.Command{
+		Use:     "add",
+		Short:   "Adds a new instances to the configuration",
+		Aliases: []string{"a"},
+		RunE: func(cmd *cobra.Command, args []string) error {
+			cfg, err := LoadConfig(rootDir)
+			if err != nil {
+				return fmt.Errorf("failed to load config %q: %w", rootDir, err)
+			}
+
+			switch nodeType {
+			case "validator":
+				for i := 0; i < count; i++ {
+					cfg = cfg.WithDigitalOceanValidator(region)
```

Comment:
[question] where does this get set? It appears to always be "" because line 15

```
	var (
		rootDir  string
		count    int
		nodeType string
		region   string
	)
```